tst/EnergyPlus/unit/HVACSizingSimulationManager.unit.cc

```diff
@@ -656,3 +658,413 @@ TEST_F(HVACSizingSimulationManagerTest, VarySysTimesteps)
 
     testSizeSimManagerObj.sizingLogger.IncrementSizingPeriodSet();
 }
+
+
+TEST_F(SQLiteFixture, HVACSizing_Passes_SQL_Output)
+{
+    // Test for #8268 - More info in the "Time" table than EnvironmentPeriods
+    // To reproduce, we need two things:
+    // * a Sizing:Plant object set to "Coincident", and,
+    // * the SimulatioNControl "Do HVAC Sizing Simulation for Sizing Periods" set to Yes
```

Comment:
That's the gist of reproducing the bug

Reply:
If it is unclear, the test below is a simple model with no zones (it's a MCVE). It has one plant loop with a WaterHeaterMixed and a pump (and a SPM scheduled) on the supply, and a LoadProfile:Plant object on the demand side.